packages/typescript-resolver-files/src/parseSources/parseSources.ts

```diff
@@ -22,7 +22,7 @@ export function parseSources(sources: Source[]): ParseSourcesResult {
       const sourcePath = path.parse(source.location);
       const moduleDir = sourcePath.dir;
 
-      const [moduleName] = moduleDir.split(path.sep).slice(-1);
+      const moduleName = path.basename(moduleDir);
```

Comment:
A similar thing is happening here- moduleDir is in this case a unix path, so "./abc/def" instead of ".\abc\def" and path.sep returns "/" on windows. so the path isn't split at all, and it is just returned in it's entirety. using path.basename() instead utilizes node's method to get the filename without the extension, which can deal with both unix and windows paths. That would have been the better option in the first place.

Reply:
`path.basename` sounds like a better implementation 👍  Thanks! @JustusFluegel